docs/docs/middlewares/httpauth.md

```diff
@@ -14,6 +15,9 @@ password    | string | ✓        | The password to be provided by user.
 httpauth:
     username: user
     password: pass
+    exclude_ip_addresses:
+        - '172.0.0.1'
```

Comment:
`172.0.0.1` looks too much like `12.0.0.1`. This could lead to misunderstandings. Please use something more normal like `77.75.79.53`